python/src/aiconfig/Config.py

```diff
@@ -35,6 +36,7 @@
 ModelParserRegistry.register_model_parser(PaLMChatParser())
 ModelParserRegistry.register_model_parser(PaLMTextParser())
 ModelParserRegistry.register_model_parser(HuggingFaceTextGenerationParser())
+ModelParserRegistry.register_model_parser(HuggingFaceTextGenerationTransformer('gpt2'))
```

Comment:
can we make this an extension? Its easy to develop here locally but this shouldn't be part of core